site/ecosystem.md

```diff
@@ -83,6 +83,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 
 ## Tools that use Vega-Lite
 
+- [prolly-yes.com](https://prolly-yes.com) An online graphical interface for probability and statistics. Leverages Vega to create beautiful and intuitive visualizations.
```

Comment:
Let's move this to the bottom of the items so wet are consistent where new things go.

Reply:
Thanks. Btw, I just changed the domain name to use subdomain, so the new domain name should be https://probability.prolly-yes.com, hope the maintainer can see this and make a change, thanks!